crates/core/component/shielded-pool/src/ics20_withdrawal.rs

```diff
@@ -118,6 +123,7 @@ impl From<Ics20Withdrawal> for pb::Ics20Withdrawal {
             timeout_time: w.timeout_time,
             source_channel: w.source_channel.to_string(),
             use_compat_address: w.use_compat_address,
+            ics20_memo: w.ics20_memo.to_string(),
```

Comment:
Do we need validation rules on memo size? I guess a user would pay gas for it, but this feels a bit scary, we do have them for governance proposals.

Reply:
There was discussion in the UIP about this, but I think that because this goes out to other chains, we really can't anticipate potential use cases, and because gas is paid, this is fine, and I think any limit that would actually be meaningfully small might be too small for some wacky use case someone comes up with.

Reply:
Sounds reasonable 👍